pyproject.toml

```
@@ -100,8 +100,8 @@ yadnav = "yadmark.navigator:launch_navigator"
 coverage = "$BROWSER htmlcov/index.html"
 test = "pytest tests"
 test-cov = ["test", "coverage"]
-lint = "pylint src/*/*.py -E"
-lint-warnings = "pylint src/*/*.py --exit-zero"
+lint = "pylint src/**/*.py -E"
+lint-warnings = "pylint src/**/*.py --exit-zero"
```

Actually, I would propose to drop the path from here all at once, and just use the configuration files.

Take into account that for `bash` they are the same, unless you enable `shopt -s globstar`. So it is shell and configuration dependent, while I believe that `src/` is just enough (and you can put in configs)

---

Or, better, you can ignore paths:
https://pylint.readthedocs.io/en/latest/user_guide/configuration/all-options.html#ignore-paths

But at a first run I would leave it empty, just drop the path here:
```suggestion
lint = "pylint -E"
lint-warnings = "pylint --exit-zero"
```

---

if you drop all path nothing happens ...

---

I confused with Pytest